projects/rero/ng-core/src/lib/record/search/record-search.component.ts

```diff
@@ -172,6 +172,11 @@ export class RecordSearchComponent implements OnInit, OnChanges, OnDestroy {
    */
   @Output() parametersChanged = new EventEmitter<any>();
 
+  /**
+   * Output current result of records by type
+   */
+  @Output() esResult = new EventEmitter<EsResult>();
```

Comment:
A better name would be `recordSearched` or something like this, it must represent an event/action.

Reply:
done